src/end-chat/end-chat-styles.tsx

```diff
@@ -18,87 +18,77 @@ import * as FlexWebChat from '@twilio/flex-webchat-ui';
 
 const { styled } = FlexWebChat;
 
-export const EndChatWrapper = styled('div')`
-  margin: 2px auto;
+export const EndChatWrapper = styled('span')`
   width: 100%;
-  display: flex;
+`;
+
+export const IconWrapper = styled('div')`
+  position: relative;
+  left: 11%;
 `;
 
 export const StyledEndButton = styled('button')`
-  margin: 5px;
-  background-color: #1876d0;
-  color: #fff;
-  width: 100%;
-  flex-grow: 1;
+  width: 90%;
+  display: flex;
+  background-color: #fff;
+  color: #d22f2f;
   height: 29px;
   font-weight: bold;
-  border: none;
+  border: 1.5px solid #d22f2f;
   border-radius: 4px;
   cursor: pointer;
   font-family: Open Sans;
-  font-size: 11px;
+  font-size: 12px;
+  margin-left: 5px;
+  height: 29px;
   line-height: 24px;
-  text-align: center;
-  white-space: nowrap;
-  overflow-wrap: break-word;
-  &:focus: {
-    outline: auto;
-  }
 `;
 
-export const ExitWrapper = styled('div')`
-  margin: 3px 2px 10px 5px;
-  width: 100%;
+export const ButtonWrapper = styled('div')`
   display: flex;
-  justify-content: space-around;
-  flex-flow: row wrap;
-`;
-
-export const ExitDescWrapper = styled('div')`
-  flex-basis: 100px;
-  flex-grow: 1;
-  display: inline-grid;
-  align-content: center;
-  justify-items: center;
+  text-align: center;
+  margin: -20px auto auto auto;
+  padding-top: 25px;
+  width: 100%;
 `;
 
 export const ExitDescText = styled('span')`
-  margin: 2px 0 0 2px;
-  color: #606b85;
-  padding: 0 10px;
+  color: #d22f2f;
+  text-align: justify;
```

Comment:
There are some viewports Exit Description text is not in place
![Screenshot 2023-04-06 at 9 36 18 AM](https://user-images.githubusercontent.com/102122005/230401183-6d5b5e00-ac14-426d-a6f4-0e46dfef9e75.png)

A simple fix I tried was changing the element to a `p` from `span` and removing `text-align: justify`

Reply:
@mythilytm thanks you the tip.